fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -325,6 +353,72 @@ class BrowserToolbarCFRPresenter(
         }
     }
 
+    @OptIn(ExperimentalComposeUiApi::class)
```

Comment:
You should remove this line @OptIn(ExperimentalComposeUiApi::class).

Reply:
Addressed!